examples/prisma-react/src/TodoList.tsx

```diff
@@ -0,0 +1,59 @@
+// @ts-ignore
+import { Todo } from '@prisma/client'
+import { useEffect, useState } from 'react'
+import NewTodo from './NewTodo'
+import { onDeleteTodo, onGetTodos, onToggleTodo } from './TodoList.telefunc'
+
+export { TodoList }
+
+function TodoItem({ refetch, ...todo }: Todo & { refetch: () => void }) {
+  return (
+    <li key={todo.id}>
+      <h2>
+        {todo.title}{' '}
+        <button
+          id={'toggle-' + todo.title}
```

Comment:
It should be `id={'toggle-' + todo.id}`. It's a no-go to use `todo.title` here and such mistake *will* cause errors in real code.